helloworld/greet/BUILD

```diff
@@ -1,28 +1,24 @@
 # Copyright 2020 Pants project contributors.
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-
 python_library(
-    # name defaults to the name of this directory, i.e., `greet`.
-    # sources defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
+    # `name` defaults to the name of this directory, i.e., `greet`.
+    # `sources` defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
     dependencies = [
         "//:translate",
         ":greetings",
         "helloworld/util"
     ],
-    compatibility = ">=3.6",
```

Comment:
This was a little confusing to have. The repo default is Py37+, and there's no need to override this to Py36+ afaict.